tools/PI/DevHome.PI/ViewModels/BarWindowViewModel.cs

```diff
@@ -277,6 +293,9 @@ private void TargetApp_PropertyChanged(object? sender, PropertyChangedEventArgs
                     ApplicationPid = process.Id;
                     ApplicationName = process.ProcessName;
                 }
+
+                // Conversely, the process chooser is only visible if we're not attached to a process
+                IsProcessChooserVisible = process is null;
```

Comment:
nit: how about we assign IsProcessChooserVisible before IsAppBarVisible and then set IsAppBarVisible = !IsProcessChooserVisible, keeping it consistent with the constructor?

Reply:
Currently I do the opposite... I'm not sure what you mean by keeping it consistent with the constructor. Can you elaborate?

        IsProcessChooserVisible = process is null;
        IsAppBarVisible = !IsProcessChooserVisible;

I did update the code, since immediately thereafter I was checking for "if (process is not null)" and it was more clear to say "if (IsAppBarVisible)"...